content/en/docs/reference/command-line-tools-reference/feature-gates/admission-webhook-match-conditions.md

```diff
@@ -12,7 +12,10 @@ stages:
     toVersion: "1.27"
   - stage: beta
     defaultValue: true
-    fromVersion: "1.28"  
+    fromVersion: "1.28"
+  - stage: stable
+    defaultValue: true
+    fromVersion: "1.30"
```

Comment:
Adding `toVersion` to the previous stage would enhance consistency with other feature gate files.

```suggestion
    fromVersion: "1.28"
    toVersion: "1.29"
  - stage: stable
    defaultValue: true
    fromVersion: "1.30"
```

Reply:
ah got it, fixed